src/components/sign-up-form/sign-up-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from 'zod'
+
+export const signUpFormSchema = z
+  .object({
+    userName: z
```

Comment:
Please change the error display for the username 
![Screenshot_23](https://github.com/flyingMerch123/internshipIncubator/assets/119298909/173e1c25-14ed-42c1-bb98-c2902aae5b9f)

Reply:
changed

Reply:
changed